src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -485,6 +485,10 @@
     ),
     PrimaryKeyGrammar=Sequence("PRIMARY", "KEY"),
     ForeignKeyGrammar=Sequence("FOREIGN", "KEY"),
+    UniqueKeyGrammar=Sequence(
+        "UNIQUE",
+        Ref.keyword("KEY", optional=True),
+    ),
```

Comment:
Is this valid in more than just MySQL? If not wouldn't it not be better to leave this as just `UNIQUE` here, and override `UniqueKeyGrammar` to include the optional `KEY` as well in MySQL?

Reply:
I've made this change to allow this to be merged and included in 0.10.0